packages/react-dom/src/events/__tests__/DOMModernPluginEventSystem-test.internal.js

```diff
@@ -1782,6 +1786,74 @@ describe('DOMModernPluginEventSystem', () => {
             dispatchClickEvent(button);
             expect(clickEvent).toHaveBeenCalledTimes(1);
           });
+
+          it('handle propagation of custom user events', () => {
+            const buttonRef = React.createRef();
+            const divRef = React.createRef();
+            const log = [];
+            const onCustomEvent = jest.fn(e =>
+              log.push(['bubble', e.currentTarget]),
+            );
+            const onCustomEventCapture = jest.fn(e =>
+              log.push(['capture', e.currentTarget]),
+            );
+
+            function Test() {
+              let custom;
```

Comment:
```suggestion
              let customEventHandle;
```
maybe not 'Handle', but I do think this should be a descriptive name